src/Mod/PartDesign/App/FeatureDraft.cpp

```diff
@@ -216,6 +219,18 @@ App::DocumentObjectExecReturn *Draft::execute()
             neutralPlane = gp_Pln(gp_Pnt(b.x, b.y, b.z), gp_Dir(n.x, n.y, n.z));
         } else if (refPlane->isDerivedFrom<App::Plane>()) {
             neutralPlane = Feature::makePlnFromPlane(refPlane);
+        } else if (refPlane->isDerivedFrom(Base::Type::fromName("Sketcher::SketchObject"))) {
+            Part::Part2DObject* sketch = static_cast<Part::Part2DObject*>(refPlane);
+            const Base::Placement& placement = sketch->Placement.getValue();
+
+            Base::Vector3d pos = placement.getPosition();
+            Base::Rotation rot = placement.getRotation();
+
+            Base::Vector3d normal(0, 0, 1);
+            rot.multVec(normal, normal);
+
+            neutralPlane = gp_Pln(gp_Pnt(pos.x, pos.y, pos.z),
+                                gp_Dir(normal.x, normal.y, normal.z));

```

Comment:
It would be worth checking if there is not a helper function (like Feature::makePlnFromPlane just above) that let you get the plane of the sketch.